R/parametricsurvivalanalysis.R

```diff
@@ -17,5 +17,2337 @@
+  summaryTable$addColumnInfo(name = "df",            title = gettext("df"),           type = "integer")
+  summaryTable$addColumnInfo(name = "aic",           title = gettext("AIC"),          type = "number", format="dp:3")
+  summaryTable$addColumnInfo(name = "bic",           title = gettext("BIC"),          type = "number", format="dp:3")
+  if (options[["modelSummaryAicWeighs"]] && length(fit) > 1)
+    summaryTable$addColumnInfo(name = "aicWeight",     title = gettext("AIC Weight"),   type = "number", format="dp:3")
+  if (options[["modelSummaryBicWeighs"]] && length(fit) > 1)
+    summaryTable$addColumnInfo(name = "bicWeight",     title = gettext("BIC Weight"),   type = "number", format="dp:3")
+  if (options[["modelSummaryRankModels"]] && length(fit) > 1)
+    summaryTable$addColumnInfo(name = "rank",        title = gettext("Rank"),         type = "integer")
+
+  if (!.saSurvivalReady(options))
+    return(summaryTable)
+
+  # extract the data
+  data <- .saSafeRbind(lapply(fit, .sapRowSummaryTable))
+
+  # add information criteria weights
+  if (options[["modelSummaryAicWeighs"]] && length(fit) > 1 && !is.null(data[[options[["modelSummaryRankModelsBy"]]]])) {
+    data$aicWeight <- .sapInformationCriteria2Weights(data[["aic"]])
+  }
+  if (options[["modelSummaryBicWeighs"]] && length(fit) > 1 && !is.null(data[[options[["modelSummaryRankModelsBy"]]]])) {
+    data$bicWeight <- .sapInformationCriteria2Weights(data[["bic"]])
+  }
+
+  # add model rank
+  if (options[["modelSummaryRankModels"]] && length(fit) > 1 && !is.null(data[[options[["modelSummaryRankModelsBy"]]]])) {
+    data <- data[order(data[[options[["modelSummaryRankModelsBy"]]]], decreasing = options[["modelSummaryRankModelsBy"]] == "logLik", na.last = TRUE), ]
+    data$rank <- seq_len(nrow(data))
+    data$rank[is.na(data$rank)] <- NA
+  }
+
+  # add footnotes
+  messages <- .sapSelectionFootnote(data, options)
+  for (i in seq_along(messages))
+    summaryTable$addFootnote(messages[[i]])
+
+  errors <- .sapCollectFitErrors(fit, options)
+  for (i in seq_along(errors))
+    summaryTable$addFootnote(errors[[i]], symbol = gettext("Error: "))
+
+  summaryTable$setData(data)
+  summaryTable$showSpecifiedColumnsOnly <- TRUE
+
+  return(summaryTable)
+}
+.sapSequentialModelComparisonTableFun    <- function(fit, options) {
+
+  # create the table
+  sequentialModelComparisonTable <- createJaspTable()
+  .sapAddColumnSubgroup(     sequentialModelComparisonTable, options, output = "coefficients")
+  .sapAddColumnDistribution( sequentialModelComparisonTable, options, output = "coefficients")
+  sequentialModelComparisonTable$addColumnInfo(name = "model0", title = gettext("H\U2080"),      type = "string")
+  sequentialModelComparisonTable$addColumnInfo(name = "model1", title = gettext("H\U2081"),      type = "string")
+  sequentialModelComparisonTable$addColumnInfo(name = "chi2",   title = gettext("\U03C7\U00B2"), type = "number")
```

Comment:
```suggestion
  sequentialModelComparisonTable$addColumnInfo(name = "model0", title = "H\U2080",      type = "string")
  sequentialModelComparisonTable$addColumnInfo(name = "model1", title = "H\U2081",      type = "string")
  sequentialModelComparisonTable$addColumnInfo(name = "chi2",   title = "\U03C7\U00B2", type = "number")
```
I wouldn't translate these. An incorrect translation could lead to an error, just try
```r
foo <- function() {
  "\U208000"
}
```
Here, R's parser already gives an error, I'm not sure how gettext would handle this.